packages/client/subscriptions/MeetingSubscription.ts

```diff
@@ -79,6 +81,7 @@ const updateHandlers = {
   SetStageTimerPayload: setStageTimerMeetingUpdater,
   ResetRetroMeetingToGroupStagePayload: resetRetroMeetingToGroupStageUpdater,
   StartDraggingReflectionPayload: startDraggingReflectionMeetingUpdater,
+  SetTaskHighlightSuccess: setTaskHighlightUpdater,
```

Comment:
can remove when isHighlighted is on the Task object

Reply:
Thanks for pointing this out and the other places which could be simplified by moving the state to the server.
Speaking for myself: When you request these bigger changes in how a problem is approached a lot of those smaller things follow automatically and are nice to figure out on my own. I think finding the balance between reducing review rounds and giving room to learn and grow is hard, but I think you could try to leave a bit more room for individual learning here.
I'm saying this because I looked at the big picture comment, reworked it accordingly and then a lot of those comments were obsolete. I wouldn't have gotten the same learning experience had I followed these comments one by one.

Reply:
great feedback! it's always a struggle to know how many parts of the same issue to flag. will limit to just 1 or 2 places in the future